infrastructure/lambda/template.yaml

```diff
@@ -516,19 +418,6 @@ Resources:
                 - 'kms:GenerateDataKey'
               Resource:
                 - !ImportValue AuditEventQueueEncryptionKeyArn
-      Events:
-        PrivateUKPassportAPIDeprecate:
-          Type: Api
-          Properties:
-            RestApiId: !Ref PrivateUKPassportAPI
-            Path: /authorization
-            Method: POST
-        PrivateUKPassportAPI:
-          Type: Api
-          Properties:
-            RestApiId: !Ref PrivateUKPassportAPI
-            Path: /check-passport
-            Method: POST
       AutoPublishAlias: live
       ProvisionedConcurrencyConfig:
         !If
```

Comment:
We shouldnt need the log group below each lambda creates there own log group by default